### PR TITLE
Updated the value to be Const

### PR DESCRIPTION
**Pull request by amitsrivastava78**
Changed the value to be const

### CODE REVIEW

**Comment by angersson** [comment]
@rthadur, I don't know very much about Toco (which is owned by the TF Lite team). Can you find a more fitting reviewer, please?

File: tensorflow/lite/toco/tooling_util.cc [hunk 1]
```diff
@@ -2330,7 +2330,7 @@ void UseArraysExtraInfo(Model* model, bool quantize_output) {
         // Make sure to create the shape even if there are no dims, to
         // correctly record 0-D shapes.
         array.mutable_shape();
-        for (int dim : entry.shape().dims()) {
+        for (const auto & dim : entry.shape().dims()) {
```
Comment (shahzadlone) [inline tensorflow/lite/toco/tooling_util.cc]: +1 for const

**Review by shahzadlone — DISMISSED** [review]

**Review by nutsiepully — DISMISSED** [review]

**Comment by amitsrivastava78** [comment]
@shahzadlone , i have rebased and merged the changes, can you pls approve again.

**Comment by amitsrivastava78** [comment]
@rthadur , i have rebased this code, and updated again, can you pls update the label to review requested.

**Review by shahzadlone — APPROVED** [review]

**Comment by amitsrivastava78** [comment]
Closing the PR as similar is already merged.

**Comment by amitsrivastava78** [comment]
Closing the PR as similar is already merged.

**Pull request closed without merging**